Userland/Libraries/LibPDF/Filter.cpp

```diff
@@ -275,7 +276,15 @@ PDFErrorOr<ByteBuffer> Filter::decode_run_length(ReadonlyBytes bytes)
     return TRY(Compress::PackBits::decode_all(bytes, OptionalNone {}, Compress::PackBits::CompatibilityMode::PDF));
 }
 
-PDFErrorOr<ByteBuffer> Filter::decode_ccitt(ReadonlyBytes, RefPtr<DictObject> decode_parms)
+static void handle_black_is_one(ByteBuffer& decoded, bool black_is_one)
```

Comment:
very nit: i'd call this "invert_bits()" and move the `if (!black_is_1)` check to the calling site.